apps/festival/festival/src/app/dashboard/analytics/title/title-analytics.component.ts

```diff
@@ -55,14 +120,46 @@ export class TitleAnalyticsComponent {
   };
   filterValue?: string;
 
+  invitations$ = this.invitationWithEventAndUserOrg().pipe(
+    joinWith({
+      analytics: () => this.getAnalyticsPerInvitation()
+    }, { shouldAwait: true }),
+    shareReplay({ bufferSize: 1, refCount: true })
+  );
+
+  ongoingScreenings$ = this.invitations$.pipe(
+    map(events => events.filter(({ event }) => eventTime(event) === 'onTime')),
+    filter(events => !!events.length),
+  );
+
+  screeningRequests$ = this.titleAnalytics$.pipe(
+    map(analytics => analytics.filter(analytic => analytic.name === 'screeningRequested')),
+    shareReplay({ bufferSize: 1, refCount: true })
+  );
+
+
+  aggregatedScreeningCards$: Observable<MetricCard[]> = combineLatest([
```

Comment:
```suggestion
  aggregatedScreeningCards$ = combineLatest([
```
It knows because `toScreenerCards` return value is typed